chainerrl/wrappers/continuing_time_limit.py

```diff
@@ -46,3 +51,76 @@ def step(self, action):
     def reset(self):
         self._elapsed_steps = 0
         return self.env.reset()
+
+
+class ContinuingTimeLimitMonitor(Monitor):
+    """`Monitor` with ChainerRL's `ContinuingTimeLimit` support.
+
+    Because of the original implementation's design,
+    explicit `close()` is needed to save the last episode.
+    Do not forget to call `close()` at the last line of your script.
```

Comment:
As I understand correctly, `gym.wrappers.Monitor`'s `close` is always called automatically by
https://github.com/openai/gym/blob/master/gym/wrappers/monitor.py#L92. Doesn't it work for our case?

Reply:
I didn't know that. I'll update the doc